waspc/src/Wasp/Generator/DbGenerator/Prisma.hs

```diff
@@ -0,0 +1,31 @@
+module Wasp.Generator.DbGenerator.Prisma where
+
+import Data.Functor ((<&>))
+import Data.List (intercalate)
+import Data.Maybe (mapMaybe)
+import qualified Wasp.AppSpec.App.Db as AS.Db
+
+-- We want to show db extensions in the following format:
+-- [first_ext, some_other(map: "name", schema: "schema", version: "1.0.0")]
+showDbExtensions :: [AS.Db.PrismaDbExtension] -> String
```

Comment:
We actually have part of our codebase dedicated to parsing and generating PSL, and this seems like it should belong there, so look for it and consider putting it there! It should be something like Wasp.PSL .

Reply:
Makes sense 👍 

I've added it to the `Wasp.Psl.Generator.Extensions` module.